package.json

```diff
@@ -32,10 +32,12 @@
     "typescript": "^4.7.4"
   },
   "dependencies": {
+    "@types/sqlite3": "^3.1.8",
```

Comment:
@arshadm this should be a devDependency instead

Reply:
you can use 

```
yarn remove @types/sqlite3 && yarn add -D @types/sqlite3
```

Reply:
Fixed.